styles/overrides/_dropdown.scss

```diff
@@ -2,17 +2,24 @@
   box-shadow: $shadow-deep;
 }
 
-.dropdown-menu > li > a {
+.dropdown-menu > li > a,
+.dropdown-menu .dropdown__link,
+.dropdown-menu .dropdown__item .dropdown__link{
```

Comment:
Normally the overrides area is used for overriding things we don't have control over. It looks like we're styling some BEM classes here. Are these styles for our components?

Reply:
They are used in our components but are being overridden by bootstrap. This enforces our own styling site wide for all dropdowns. I mean this could be moved to the app but I felt it would just be replicating the style we have here. Thoughts ? @kyleharper

Reply:
That makes sense. The only thing that doesn't is I thought `.dropdown-menu > li > a` covered what we required? 

If the same styling is applied to both `.dropdown-menu > li > a` and the BEM related classes then I recommend duplicating the styles into the component. You could even use a `mixin` for the styles if you want it DRYer.

The reason why I suggest this is to keep the overrides responsible for overriding 3rd party libraries. If we start crossing the lines, we'll have issues later on I think.